### PR TITLE
IGP Oracle Updates

### PR DESCRIPTION
**Pull request by nambrot**
### Description

Had a bit of back and forth on updating the oracles. Executed the config for EVM <> EVM routes. Had to effectively disable StorageGasOracle setting because of https://github.com/hyperlane-xyz/issues/issues/1178. So this probably shouldn't be merged as is, but wanted to open it up. Should be considred up-to-date vs. https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3502

### CODE REVIEW

File: typescript/sdk/src/gas/HyperlaneIgpDeployer.ts [hunk 1]
```diff
@@ -95,7 +96,7 @@ export class HyperlaneIgpDeployer extends HyperlaneDeployer<
       return gasOracle;
     }
 
-    this.logger.debug(`Configuring gas oracle from ${chain}...`);
+    console.log(`Configuring gas oracle from ${chain}...`);
```
Comment (nbayindirli) [inline typescript/sdk/src/gas/HyperlaneIgpDeployer.ts:99]: Why switch back from `logger`?

**Review by nbayindirli — APPROVED** [review]
LGTM aside from minor comment on `logger`

File: typescript/sdk/src/gas/HyperlaneIgpDeployer.ts (hunk 1 above)
Comment (nbayindirli) [reply typescript/sdk/src/gas/HyperlaneIgpDeployer.ts:99]: Nvm– see this likely won't be merged

Comment (jmrossy) [reply typescript/sdk/src/gas/HyperlaneIgpDeployer.ts:99]: +1, should be reverted

File: typescript/infra/config/environments/mainnet3/gasPrices.json [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-  "ethereum": "26.346912847",
-  "mantapacific": "0.100000057",
-  "moonbeam": "125.0",
-  "optimism": "0.003225814",
-  "polygon": "61.601287856",
-  "gnosis": "1.852997796",
-  "base": "0.0010003",
-  "scroll": "0.46",
-  "polygonzkevm": "3.95",
+  "bsc": "7",
```
Comment (tkporter) [inline typescript/infra/config/environments/mainnet3/gasPrices.json:4]: did you update just gasPrices or also token exchange rates?

also are these values handcrafted or were they generated by the tooling?

Do you see yourself merging this into main or that https://github.com/hyperlane-xyz/issues/issues/1178 will take this over / supersede this

**Pull request closed without merging**